internal/client/gitea.go

```diff
@@ -0,0 +1,161 @@
+package client
+
+import (
+	"os"
+	"strconv"
+
+	"code.gitea.io/gitea/modules/structs"
+	"code.gitea.io/sdk/gitea"
+	"github.com/apex/log"
+	"github.com/goreleaser/goreleaser/internal/artifact"
+	"github.com/goreleaser/goreleaser/internal/tmpl"
+	"github.com/goreleaser/goreleaser/pkg/config"
+	"github.com/goreleaser/goreleaser/pkg/context"
+)
+
+type giteaClient struct {
+	client *gitea.Client
+}
+
+// NewGitea returns a gitea client implementation
+func NewGitea(ctx *context.Context) (Client, error) {
+	token := ctx.Token
+	client := gitea.NewClient(ctx.Config.Release.Gitea.InstanceURL, token)
```

Comment:
is there also an option for `SkipTlsVerify` like the other clients have?

Reply:
Maybe if we use https://godoc.org/code.gitea.io/sdk/gitea#NewClientWithHTTP

Reply:
Yes, that's what I meant